types/core/config.d.ts

```diff
@@ -1,141 +1,1045 @@
+      TOKEN_CONFIGURE: CONFIG.Permission;
+      WALL_DOORS: CONFIG.Permission;
+    } & Partial<Record<string, CONFIG.Permission>>;
+  };
+
+  /**
+   * Configure the default Token text style so that it may be reused and overridden by modules
+   * @defaultValue
+   * ```
+   * new PIXI.TextStyle({
+   *   fontFamily: 'Signika',
+   *   fontSize: 36,
+   *   fill: '#FFFFFF',
+   *   stroke: '#111111',
+   *   strokeThickness: 1,
+   *   dropShadow: true,
+   *   dropShadowColor: '#000000',
+   *   dropShadowBlur: 4,
+   *   dropShadowAngle: 0,
+   *   dropShadowDistance: 0,
+   *   align: 'center',
+   *   wordWrap: false,
+   *   padding: 1
+   * })
+   * ```
+   **/
+  canvasTextStyle: PIXI.TextStyle;
+
   /**
    * The control icons used for rendering common HUD operations
    */
   controlIcons: {
-    [key: string]: string;
+    /**
+     * @defaultValue `'icons/svg/combat.svg'`
+     */
     combat: string;
-    defeated: string;
-    down: string;
+
+    /**
+     * @defaultValue `'icons/svg/cowled.svg'`
+     */
+    visibility: string;
+
+    /**
+     * @defaultValue `'icons/svg/aura.svg'`
+     */
     effects: string;
+
+    /**
+     * @defaultValue `'icons/svg/padlock.svg'`
+     */
     lock: string;
+
+    /**
+     * @defaultValue `'icons/svg/up.svg'`
+     */
     up: string;
-    visibility: string;
-  };
+
+    /**
+     * @defaultValue `'icons/svg/down.svg'`
+     */
+    down: string;
+
+    /**
+     * @defaultValue `'icons/svg/skull.svg'`
+     */
+    defeated: string;
+
+    /**
+     * @defaultValue `'icons/svg/fire.svg'`
+     */
+    light: string;
+
+    /**
+     * @defaultValue `'icons/svg/explosion.svg'`
+     */
+    template: string;
+
+    /**
+     * @defaultValue `'icons/svg/sound.svg'`
+     */
+    sound: string;
+
+    /**
+     * @defaultValue `'icons/svg/door-steel.svg'`
+     */
+    doorClosed: string;
+
+    /**
+     * @defaultValue `'icons/svg/door-exit.svg'`
+     */
+    doorOpen: string;
+
+    /**
+     * @defaultValue `'icons/svg/padlock.svg'`
+     */
+    doorLocked: string;
+  } & Partial<Record<string, string>>;
 
   /**
-   * Configure debugging flags to display additional information
+   * Suggested font families that are displayed wherever a choice is presented
+   * @defaultValue `['Arial', 'Arial Black', 'Comic Sans MS', 'Courier New', 'Times New Roman', 'Signika', 'Modesto Condensed']`
    */
-  debug: {
-    hooks: boolean;
-    sight: boolean;
-  };
+  fontFamilies: string[];
 
   /**
    * The default font family used for text labels on the PIXI Canvas
+   * @defaultValue `'Signika'`
    */
   defaultFontFamily: string;
 
   /**
-   * Suggested font families that are displayed wherever a choice is presented
-   */
-  fontFamilies: string[];
-
-  /**
-   * Maximum canvas zoom scale
+   * An array of status effect icons which can be applied to Tokens
    */
-  maxCanvasZoom: number;
+  statusEffects: { id: string; label: string; icon: string }[];
```

Comment:
I think this should have a documented default value.

Reply:
The default value is
```js
[
    {
      id: "dead",
      label: "EFFECT.StatusDead",
      icon: "icons/svg/skull.svg"
    },
    {
      id: "unconscious",
      label: "EFFECT.StatusUnconscious",
      icon: "icons/svg/unconscious.svg"
    },
    {
      id: "sleep",
      label: "EFFECT.StatusAsleep",
      icon: "icons/svg/sleep.svg"
    },
    {
      id: "stun",
      label: "EFFECT.StatusStunned",
      icon: "icons/svg/daze.svg"
    },
    {
      id: "prone",
      label: "EFFECT.StatusProne",
      icon: "icons/svg/falling.svg"
    },
    {
      id: "restrain",
      label: "EFFECT.StatusRestrained",
      icon: "icons/svg/net.svg",
    },
    {
      id: "paralysis",
      label: "EFFECT.StatusParalysis",
      icon: "icons/svg/paralysis.svg",
    },
    {
      id: "fly",
      label: "EFFECT.StatusFlying",
      icon: "icons/svg/wing.svg",
    },
    {
      id: "blind",
      label: "EFFECT.StatusBlind",
      icon: "icons/svg/blind.svg"
    },
    {
      id: "deaf",
      label: "EFFECT.StatusDeaf",
      icon: "icons/svg/deaf.svg"
    },
    {
      id: "silence",
      label: "EFFECT.StatusSilenced",
      icon: "icons/svg/silenced.svg"
    },
    {
      id: "fear",
      label: "EFFECT.StatusFear",
      icon: "icons/svg/terror.svg"
    },
    {
      id: "burning",
      label: "EFFECT.StatusBurning",
      icon: "icons/svg/fire.svg"
    },
    {
      id: "frozen",
      label: "EFFECT.StatusFrozen",
      icon: "icons/svg/frozen.svg"
    },
    {
      id: "shock",
      label: "EFFECT.StatusShocked",
      icon: "icons/svg/lightning.svg"
    },
    {
      id: "corrode",
      label: "EFFECT.StatusCorrode",
      icon: "icons/svg/acid.svg"
    },
    {
      id: "bleeding",
      label: "EFFECT.StatusBleeding",
      icon: "icons/svg/blood.svg"
    },
    {
      id: "disease",
      label: "EFFECT.StatusDisease",
      icon: "icons/svg/biohazard.svg"
    },
    {
      id: "poison",
      label: "EFFECT.StatusPoison",
      icon: "icons/svg/poison.svg"
    },
    {
      id: "radiation",
      label: "EFFECT.StatusRadiation",
      icon: "icons/svg/radiation.svg"
    },
    {
      id: "regen",
      label: "EFFECT.StatusRegen",
      icon: "icons/svg/regen.svg"
    },
    {
      id: "degen",
      label: "EFFECT.StatusDegen",
      icon: "icons/svg/degen.svg"
    },
    {
      id: "upgrade",
      label: "EFFECT.StatusUpgrade",
      icon: "icons/svg/upgrade.svg"
    },
    {
      id: "downgrade",
      label: "EFFECT.StatusDowngrade",
      icon: "icons/svg/downgrade.svg"
    },
    {
      id: "target",
      label: "EFFECT.StatusTarget",
      icon: "icons/svg/target.svg"
    },
    {
      id: "eye",
      label: "EFFECT.StatusMarked",
      icon: "icons/svg/eye.svg"
    },
    {
      id: "curse",
      label: "EFFECT.StatusCursed",
      icon: "icons/svg/sun.svg"
    },
    {
      id: "bless",
      label: "EFFECT.StatusBlessed",
      icon: "icons/svg/angel.svg"
    },
    {
      id: "fireShield",
      label: "EFFECT.StatusFireShield",
      icon: "icons/svg/fire-shield.svg"
    },
    {
      id: "coldShield",
      label: "EFFECT.StatusIceShield",
      icon: "icons/svg/ice-shield.svg"
    },
    {
      id: "magicShield",
      label: "EFFECT.StatusMagicShield",
      icon: "icons/svg/mage-shield.svg"
    },
    {
      id: "holyShield",
      label: "EFFECT.StatusHolyShield",
      icon: "icons/svg/holy-shield.svg"
    },
  ]
  ```
  That's really long, do you really think that should be in the TSDoc?

Reply:
done